tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -609,9 +615,17 @@ def _request(self, method, obj_type, *args, **kwargs):
         request = getattr(request_factory, method)(*args, **kwargs)
 
         response = request.submit()
+
+        # Check the result status and accordingly set the error state
+        if response['status'] != 0:
```

Comment:
Because of this condition  create pool flow is failing, because response from ceph does not have status field, response structure of rados_command and rbd_command is different. To solve the we have to modify the if condition as we discussed.
if ('status' in response):
            if response['status'] != 0:
                request.error = True
                request.error_message = response['err']
elif response['error']:
                request.error = True
                request.error_message = response['error_status']